src/inc/corhdr.h

```diff
@@ -913,6 +913,9 @@ typedef enum CorElementType
     ELEMENT_TYPE_SENTINEL       = 0x01 | ELEMENT_TYPE_MODIFIER, // sentinel for varargs
     ELEMENT_TYPE_PINNED         = 0x05 | ELEMENT_TYPE_MODIFIER,
 
+    ELEMENT_TYPE_SHORT_VECTOR   = 0x80,
```

Comment:
This file is meant to contain only the standard `ELEMENT_TYPE_` constants that show up in regular metadata.

The various custom area specific should rather be somewhere else. corpriv.h may be a good place - there are several of them already (look for `ELEMENT_TYPE_`).